apps/prairielearn/src/middlewares/authzCourseOrInstance.ts

```diff
@@ -120,7 +122,11 @@ export async function authzCourseOrInstance(req, res) {
   res.locals.use_bootstrap_4 = await features.enabledFromLocals('bootstrap-4', res.locals);
 
   // Check if it is necessary to request a user data override - if not, return
-  let overrides = [];
+  const overrides: {
+    name: string;
+    value: string;
+    cookie: string;
+  }[] = [];
```

Comment:
Nit, consider defining an `Override` type or the like and using it here and in `clearOverrideCookies`.